### DISCUSSION TITLE
Feature request: Ability to add specific folders or files within a repo to Favorites (GitHub Mobile)

### DISCUSSION THREAD

**Discussion by erbanku** [community/community#43278] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Dear GitHub team,

I use the Git Repo to store and organize my notes, and I regularly push my updates to a GitHub Private Repo for backup and easy access on my mobile device using the GitHub Mobile app. 
However, I have run into a problem where **I cannot add specific folders or files within my Private Repo to my Favorites in the GitHub Mobile app**, making it inconvenient to access specific notes folders (I organize my notes by month, month as the folder name) quickly. I can achieve this on the GitHub website by bookmarking the folder address, but it would be **much more convenient** for mobile users if it were also implemented in the GitHub Mobile app.
So, I'm asking for a feature that **allows users to add specific folders or files within a Repo to their Favorites in the GitHub Mobile app**, similar to how it can be done by adding a repo to favorites on the GitHub Mobile app. 
This would significantly improve the usability and convenience of the GitHub Mobile app for users like myself who rely on it for quick access to their notes and other important files or folders.

Thank you very much for considering this request.

**Comment by HenSquared** [comment, marked as answer]
Hey @erbanku I can understand wanting to bookmark things on the Mobile App! Thanks for the feedback. I think it would be great to be able to easily access these items in the app through favorites or shortcuts! 

I'll add this to our internal feedback tracking so the team can discuss!
